package.json

```diff
@@ -2,7 +2,7 @@
   "name": "eruption",
   "description": "Next Generation Boilerplate for React/Typescript, built on top of Vite ⚡️",
   "private": false,
-  "version": "1.6.0",
+  "version": "1.7.0",
```

Comment:
Updated here just to mach with the [latest release](https://github.com/eruptionjs/core/releases/tag/v1.7).